lib/support/chrome-perflog-parser.js

```diff
@@ -297,6 +302,12 @@ module.exports = {
       }
     }
 
+    entries.sort((a,b) => {
```

Comment:
Good catch I was using this to sort the entries by the requestIds. They were out of order causing the waterfall to not look right. I'll update to use the requestIds.